PcdcAnalysisTools/api.py

```diff
@@ -103,8 +102,8 @@ def app_init(app):
 
 
 app = Flask(__name__)
-
-
+if os.environ['MOCK_DATA'] == 'True':
+    app_register_blueprints(app)
```

Comment:
@paulmurdoch19 why did you ad this?

Reply:
I did not have the correct setup data so when the app_init  function runs it will fail on app.config["RSA_PRIVATE_KEY"] = SignatureManager(key_path=key_path).get_key() and never run  app_register_blueprints(app) so I wouldnt be able to access the endpoints